markdown_stoplight/changelog.md

```diff
@@ -2,6 +2,40 @@
 
 This change log covers significant updates and changes to the Beyond REST API.
 
+**2019-05-22**
+
+**New**
+
+* [Enable purchasability for variation](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/products/variation-availability-enable-purchasability)
+
+* [Disable purchasability for variation](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/products/variation-availability-disable-purchasability)
+
+* [Show variation availability details](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/products/variation-availability-get).
+
+* [Adjust stock level of variation](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/products/variation-availability-adjust-available-stock).
+
+**Update**
+
+* Added `sendMail` property to [Create return process](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/orders/create-return-process).
```

Comment:
Maybe add a sentence for the default value (false, afaik), if not set on request.